### PR TITLE
Tenant wallets

### PR DESCRIPTION
**Pull request by elv-wayne**
Returns the wallets associated with the tenant and marketplace (not sure if we need marketplace but the elvauthd api needs it)

`./elv-live tenant_wallets itenYQbgk66W1BFEqWr95xPmHZEjmdF iq__2FrA2S1XBy4zdRGQn1knakpbrBV4 --max_results 3`

```
Tenant wallets tenant: itenYQbgk66W1BFEqWr95xPmHZEjmdF marketplace: iq__2FrA2S1XBy4zdRGQn1knakpbrBV4 max_results: 3
Network: demov3
contents:
  - addr: '0x6dd34d44e39aa31bacaa1fb62d470775c6a897e3'
    ident: christopher.campbell@eluv.io
    extra_json:
      share_email: true
  - addr: '0x01abfdeabe5e9842a696ee77f6c048f6df467ef2'
    extra_json:
      consent: true
  - addr: '0xbda76f44f7070f0519a7bac520f24050e1d6c64e'
    ident: jenya.armen@eluv.io
    extra_json:
      consent: true
paging:
  start: 0
  limit: 3
  total: 157
```

### CODE REVIEW

**Review by elv-serban — COMMENTED** [review]
@elv-wayne can you pull the 'list' PR and then update this branch to latest main? That will reduce this diff.

**Comment by elv-wayne** [comment]
@elv-serban this branch has all the ones from 'list'. I merged 'list' into main but it still shows large diffs. If you look at this commit only, it's essentially only these changes:

https://github.com/eluv-io/elv-live-js/pull/13/commits/e5523f65ffb763fc94a4ee6cbcec98ae4bab5460

File: src/EluvioLive.js [hunk 1]
```diff
@@ -1268,14 +1269,17 @@ Lookup NFT: https://wallet.contentfabric.io/lookup/`;
     return { signature, multiSig };
   }
 
-  async MakeAuthServiceRequest({ method, path, body }) {
-    const { signature, multiSig } = await this.Sign({
+  async PostServiceRequest({path, body }) {
```
Comment (elv-serban) [inline src/EluvioLive.js:1272]: this is specifically for the 'authority' service (which is the only one with the 'signed body' auth)
didn't you want to have it say AuthorityServiceRequest?

Maybe PostAuthServiceRequest

**Review by elv-serban — APPROVED** [review]
Looks good - minor recommendation on function name

Also style/tabs seem off.

File: src/EluvioLive.js (hunk 1 above)
Comment (elv-wayne) [reply src/EluvioLive.js:1272]: Will organize it later into it's own classes so didn't want to make too much of a distinction. I was also assuming all our Services would have the same consistent auth procedure.

**Pull request merged**